models/model.js

```diff
@@ -44,3 +44,25 @@ exports.fetchCommentsByReviewID = (review_id) => {
       }
     });
 };
+
+exports.insertCommentsByReviewID = (newComment, review_id) => {
+  return db
+    .query("SELECT * FROM users WHERE username = $1", [newComment.username])
+    .then((array) => {
+      if (array.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "invalid username" });
+      } else {
```

Comment:
Sql will throw an error if you try to insert a comment with an invalid username anyway, so you could just handle this in a single query with an error handler